src/components/Blog/CustomRichBlocks.tsx

```diff
@@ -214,7 +225,7 @@ export const CustomRichBlocks = ({
                         strikethrough={el.props.strikethrough}
                         underline={el.props.underline}
                         bold={el.props.bold}
-                        key={`blog-paragraph-line-${index}-${lineIndex}`}
+                        key={generateKey('blog-paragraph-line')}
```

Comment:
This key might conflicts with another one as it does not have any variable in anymore

Reply:
Thanks Sebastien! I have undone those changes - but out of curiosity: Does the generateKey not create a random key anyways, already preventing a clash with any existing key?

Reply:
Hum, you are actually right